### ISSUE TITLE
[REQUEST] Remember position & zoom (cookie?)

### ISSUE THREAD

**Issue by gideonfisk** [favll/pogom#324]
## Checklist
 - [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
 - [x] I have checked if this issue already exists (both open and closed issues)
 - [x] I am using the most recent version of pogom from the master branch
 - [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
 - [ ] I'm posting the complete log
 - [x] I'm using a descriptive title for this issue

## My Issue
It would be very convenient if the web page remembered your last position (location) and zoom level. This could probably be stored in a cookie or some other session-level item.
...

**Comment by MinnesotaMiracleMan** [comment]
You can edit the zoom level it as is. The position is a little tougher, as you have to set the initial location scanned as the one you want it centered on.

**Comment by MinnesotaMiracleMan** [comment]
"You can edit the zoom level it as is. The position is a little tougher, as you have to set the initial location scanned as the one you want it centered on."

I should add that I believe the file the zoom level is located in is map.jd in the \static folder. 

The location centering is a little tougher as you have to change the order of locations in the config file, and despite having one first in the list it was not always the first scanned for me, and thus didn't always get centered correctly.

**Comment by gideonfisk** [comment]
Yeah but that's not quite what I'm talking about. I'm saying when user X opens the webpage we should remember where they had panned to and how far they had zoomed in. 

At least as an option.
